### PR TITLE
Docker chores

### PR DESCRIPTION
**Pull request by elgalu**
* Fix README.md typos
* Dockerfile minor improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Dockerised Pact Broker
-==================
+Dockerized Pact Broker
+======================
```
Comment (bethesque) [inline README.md]: We're Australian, we use "s" :P

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 RUN rm -f /etc/service/nginx/down
 RUN rm /etc/nginx/sites-enabled/default
-ADD container /
 
-ADD pact_broker/Gemfile $APP_HOME/
-ADD pact_broker/Gemfile.lock $APP_HOME/
+ADD container /
+ADD pact_broker $APP_HOME
 RUN chown -R app:app $APP_HOME
-RUN su app -c "cd $APP_HOME && bundle install --deployment --without='development test'"
```
Comment (bethesque) [inline Dockerfile]: Separating the files is deliberate, to allow changes to be made to later files without having to rebuild all the layers.

File: README.md (hunk 1 above)
Comment (elgalu) [inline README.md]: =) fixed and squashed in 1 commit.

File: Dockerfile (hunk 2 above)
Comment (elgalu) [inline Dockerfile]: fixed and squashed in 1 commit

**Comment by bethesque** [comment]
I'd like to get the configurable port PR in first.

**Comment by elgalu** [comment]
I'll close this as it doesn't help.
Typos moved to https://github.com/DiUS/pact_broker-docker/pull/18

**Pull request closed without merging**